src/action.ts

```diff
@@ -231,14 +231,42 @@ export const runAction = async (space): Promise<void> => {
     Logger.verbose("No alias changes required");
   }
 
+  // "closed" action happens on PR close and PR merge
+  const githubAction = github.context.payload.action
+
+  // If CDA token is created and we want to purge ephemeral env upon close/merge
+  // then ephemeral token should be deleted as well
+  if ( 
+    DELETE_FEATURE && 
+    CREATE_CDA_TOKEN && 
+    githubAction === "closed" 
+  ) {
+    Logger.verbose(`debug: attempting to delete ${tokenKeyName}`)
+
+    const { items: keys } = await space.getApiKeys();
+    const k = keys.find(key => key.name === tokenKeyName)
+
+    if (k === undefined) {
+        Logger.warn(`could not find ephemeral toke ${tokenKeyName}, possibly it was deleted manually`);
+    } else {
+      try {
+        await k.delete();
+        Logger.success(`removed ephemeral token ${tokenKeyName}`);
+      } catch(error) {
+        Logger.error("Unable to delete ephemeral token");
+        Logger.error(error);
```

Comment:
should the additional error messaging `warn` or `info` rather than error so it gets shown or not based on a `--verbose` flag?